sycl/test-e2e/Graph/Inputs/multiple_kernel_bundles.cpp

```diff
@@ -0,0 +1,119 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Tests using a bundle in a graph.
+// Checks the PI call trace to ensure that the bundle kernel of the single task
+// is used.
+
+#include "../graph_common.hpp"
+
+class Kernel1Name;
+class Kernel2Name;
+
+int main() {
+  using T = unsigned short;
```

Comment:
should we make this `int` so that all the buffers use the same type, `int` is what `Buf1` & `Buf2` are using